src/views/List/List.css

```diff
@@ -0,0 +1,25 @@
+.div {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  justify-content: flex-start;
+  height: calc(100vh - 75px);
```

Comment:
Nice use and comfortability with the calc function here! ✨ There's a flex-end property on the body element that's causing all of the content to go to the bottom of the page and this is an interesting solution to get the layout you want